src/contract_wrappers/exchange_wrapper.ts

```diff
@@ -207,7 +220,10 @@ export class ExchangeWrapper extends ContractWrapper {
                                      shouldCheckTransfer: boolean, takerAddress: string): Promise<BigNumber.BigNumber> {
         const takerTokenAddresses = _.map(signedOrders, signedOrder => signedOrder.takerTokenAddress);
```

Comment:
We need `assert.doesConformToSchema('signedOrders', signedOrders, signedOrdersSchema);` assertion moved up before this line.